Update microsoft-staffhub-to-be-retired.md (#2472)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/expand-teams-across-your-org/shifts/microsoft-staffhub-to-be-retired.md

#### When will StaffHub be retired?

> Starting the middle of April 2019, Microsoft StaffHub will no longer be available for new tenants and you will no longer be able to access the app from the Office 365 app launcher in existing tenants. Current users will still be able to use the service until October 1, 2019, however no additional features will be made available.

Reviewer: ```suggestion
Since the middle of April, 2019, Microsoft StaffHub is not available for new tenants and you can no longer access the app from the Office 365 app launcher in existing tenants. Current users can still use the service until October 1, 2019, however no additional features will be made available.
```

#### Where can I provide feedback?
  

> In the mobile app, tap **More** in the upper left of the screen, go to **Settings** > **Send feedback** or **Report an issue**. Similarly, in the desktop and web apps, in the lower left of the screen, go **Help** > **Share an idea** or **Report an issue**.

Reviewer: ```suggestion
In the mobile app, tap **More** in the upper left of the screen, go to **Settings** > **Send feedback** or **Report an issue**. Similarly, in the desktop and web apps, in the lower left of the screen, go to **Help** > **Share an idea** or **Report an issue**.
```
